### DISCUSSION TITLE
How can I change the way that webserver url is formed.

### DISCUSSION THREAD

**Discussion by Yashar78** [apache/airflow#37479] [General]
I am in a cluster where the urls looks like http://blabla/user/xx/proxy/8501/ . This is for the application named app_name, user xx and port 8501. But when I run airflow webserver it attaches the port number to the baseurl and craetes a url like http://base_url:port_number which obviously is not reachable. Where in the source code I should modify to create webserver urls like what I need ?

**Comment by andresitorusz** [comment]
You can customize the Webserver url using this environment variable -> [AIRFLOW__WEBSERVER__BASE_URL](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#base-url).

**Comment by Yashar78** [comment]
I get this error "Apache Airflow is not at this location" when opening the webserver url. Should I change any other parameter?
